src/components/workplaceExamples.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import {
+  workplaceExamples,
+  workplaceExamplesHeader,
+} from './workplaceExamples.module.css'
+import WorkplaceExamplesIconSvg from '../assets/icons/workplaceExamplesIcon.svg'
+
+export default function WorkplaceExamples({ title, body }) {
```

Comment:
this will take 4 pieces of data (4 workplace examples) but the title will always be "Workplace Examples"